### PR TITLE
Move hero-block to separate element. Add dates

### PR DESCRIPTION
**Pull request by ozasadnyy**

### CODE REVIEW

File: src/elements/hero-block.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    <app-toolbar class="hero-block" layout start vertical center-justified>
+      <plastic-image
+        class="hero-image"
+        srcset="[[hero.background.image]]"
+        sizing="cover"
+        lazy-load preload fade fit
+      ></plastic-image>
+      <div class="hero-overlay" show$=[[hero.background.image]] fit></div>
+      <div class="container">
+        <iron-pages selected="[[route.route]]" attr-for-selected="route" fallback-selection="default">
```
Comment (sophieH29) [inline src/elements/hero-block.html]: selected$ to avoid problems on safari

File: src/elements/hero-block.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+          margin-top: 0;
+        }
+      }
+
+    </style>
+
+    <app-toolbar class="hero-block" layout start vertical center-justified>
+      <plastic-image
+        class="hero-image"
+        srcset="[[hero.background.image]]"
```
Comment (sophieH29) [inline src/elements/hero-block.html]: srcset$ to avoid problems on safari

File: src/hoverboard-app.html [hunk 3]
```diff
@@ -383,25 +322,7 @@ <h3 class="location">{$ location.short $}</h3>
             </paper-menu-button>
           </app-toolbar>
 
-          <app-toolbar class="hero-block" layout start vertical center-justified>
-            <plastic-image
-              class="hero-image"
-              srcset="[[hero.background.image]]"
```
Comment (sophieH29) [inline src/hoverboard-app.html]: I don't see you've removed this from properties
   hero: {
            type: Object,
            statePath: 'hero',
            observer: '_heroChanged'
          },

haven't you?

File: src/elements/hero-block.html (hunk 2 above)
Comment (ozasadnyy) [reply src/elements/hero-block.html]: no, it's polymer property

File: src/elements/hero-block.html (hunk 1 above)
Comment (ozasadnyy) [reply src/elements/hero-block.html]: no, it's polymer property

File: src/elements/hero-block.html (hunk 2 above)
Comment (sophieH29) [reply src/elements/hero-block.html]: but sometimes you use it, and sometimes you don't

http://take.ms/m9yNO

Comment (sophieH29) [reply src/elements/hero-block.html]: or you mean srcset is a property and we don't need this, hidden is an attribute, correct?

Comment (ozasadnyy) [reply src/elements/hero-block.html]: Because sometimes we need to have attribute in a DOM, sometimes just pass data to the polymer element

Comment (sophieH29) [reply src/elements/hero-block.html]: got it

**Pull request merged**